server.js

```diff
@@ -267,33 +267,41 @@ app.post('/slambook', async (req, res) => {
 //   }
 // });
 
-// app.put('/slambook/:id', async (req, res) => {
-//   try {
-//     const updatedEntry = await SlamBook.findByIdAndUpdate(req.params.id, req.body, { new: true });
-//     if (updatedEntry) {
-//       res.json({ message: 'Entry Updated' });
-//     } else {
-//       res.status(404).json({ message: 'Entry not found' });
-//     }
-//   } catch (error) {
-//     console.error('Error updating slambook entry:', error);
-//     res.status(500).json({ error: 'Internal Server Error' });
-//   }
-// });
+app.put('/slambook/:id', async (req, res) => {
+  try {
+    const updatedEntry = await SlamBook.findByIdAndUpdate(req.params.id, req.body, { new: true });
+    if (updatedEntry) {
+      res.json({ message: 'Entry Updated' });
+    } else {
+      res.status(404).json({ message: 'Entry not found' });
+    }
+  } catch (error) {
+    console.error('Error updating slambook entry:', error);
+    res.status(500).json({ error: 'Internal Server Error' });
+  }
+});
+
```

Comment:
It is recommended to log the error stack trace (error.stack) in addition to the error message to gain more information about the exact cause of the error. This can be helpful for debugging purposes.